client/verta/verta/_deployment/endpoint.py

```diff
@@ -470,3 +476,48 @@ def get_update_status(self):
         response = _utils.make_request("GET", url, self._conn)
         _utils.raise_for_http_error(response)
         return response.json()
+
+    def download_manifest(
+            self, download_to_path, name, strategy,
+            resources=None, autoscaling=None, env_vars=None):
+        """
+        Downloads this Endpoint's Kubernetes manifest YAML.
+
+        Parameters
+        ----------
+        download_to_path : str
+            Path to download manifest YAML to.
+        name : str
+            Name of the Endpoint.
+        strategy : :class:`~verta.deployment.update._strategies._UpdateStrategy`
+            Strategy (direct or canary) for updating the Endpoint.
+        resources : list of :class:`~verta.deployment.resources._Resource`, optional
+            Resources allowed for the updated Endpoint.
+        autoscaling : :class:`~verta.deployment.autoscaling._autoscaling.Autoscaling`, optional
+            Autoscaling condition for the updated Endpoint.
+        env_vars : dict of str to str, optional
+            Environment variables.
+
+        Returns
+        -------
+        downloaded_to_path : str
+            Absolute path where deployment YAML was downloaded to. Matches `download_to_path`.
+
+        """
+        data = {
+            'endpoint': {'path': self.path},
```

Comment:
Downloading the manifest should not require the endpoint to already exist before.

Reply:
@conradoverta 
Shoot, really? Hrm the code's not going to work as a static method because it needs the `._conn` of an object instance. The best I can do right now would be a `client.download_endpoint_manifest()`. How does that sound?

Reply:
Makes sense to me